integrations/clickup/clickup/client.py

```diff
@@ -0,0 +1,172 @@
+from loguru import logger
+from typing import Any, AsyncGenerator
+
+from port_ocean.utils import http_async_client
+
+from .utils import timestamp_to_datetime
+
+
+WEBHOOK_NAME = "Port-Ocean-Events-Webhook"
+WEBHOOK_EVENTS = [
+    "listCreated",
+    "listUpdated",
+    "listDeleted",
+    "taskCreated",
+    "taskUpdated",
+    "taskDeleted",
+    "taskCommentPosted",
+    "taskCommentUpdated",
+]
+
+
+class ClickupClient:
+    def __init__(self, clickup_personal_token: str) -> None:
+        self.clickup_url = "https://api.clickup.com/api/v2"
+        self.clickup_personal_token = clickup_personal_token
+
+        self.api_auth_header = {"Authorization": self.clickup_personal_token}
+        self.client = http_async_client
+        self.client.headers.update(self.api_auth_header)
+
+    async def create_events_webhook(self, app_host: str) -> None:
+        teams = self.get_paginated_teams()
+        async for team_batch in teams:
```

Comment:
since get_paginated_teams() is an async generator, you could use it directly in the async for loop as

async for team_batch in self.get_paginated_teams():